setup.py

```diff
@@ -22,7 +23,7 @@
 
 # Get some values from the setup.cfg
 from distutils import config
-conf = config.ConfigParser()
+conf = configparser.ConfigParser()
 conf.read(['setup.cfg'])
```

Comment:
The standard way to do this, that I've seen in many Astropy packages and I implemented in some of my codes, is the following:

```
# Get some values from the setup.cfg
try:
    from ConfigParser import ConfigParser
except ImportError:
    from configparser import ConfigParser

conf = ConfigParser()
```
No need for `from distutils import config` anymore!